packages/sdk/tests/e2e/dataProtectorSharing/consumeProtectedData.test.ts

```diff
@@ -1,65 +1,46 @@
-import { beforeAll, describe, expect, it, jest } from '@jest/globals';
+import { beforeAll, describe, expect, it } from '@jest/globals';
 import { Wallet } from 'ethers';
-import { utils } from 'iexec';
 import { IExecDataProtector } from '../../../src/index.js';
-import { timeouts } from '../../test-utils.js';
-
-const WORKERPOOL_ADDRESS = 'prod-stagingv8.main.pools.iexec.eth';
+import {
+  MAX_EXPECTED_MARKET_API_PURGE_TIME,
+  TEST_CHAIN,
+  createAndPublishWorkerpoolOrder,
+  getTestConfig,
+  timeouts,
+} from '../../test-utils.js';
+import { WORKERPOOL_ADDRESS } from '../../../src/config/config.js';
+
+const DEFAULT_PROTECTED_DATA_DELIVERY_APP =
+  '0x85795d8Eb2B5D39A6E8dfB7890924191B3D1Ccf6';
 
 // this test can't run in drone ci need VPN because is based on iexec protocol, unskip it when desired
-describe.skip('dataProtector.consumeProtectedData()', () => {
+describe('dataProtector.consumeProtectedData()', () => {
   let dataProtectorCreator: IExecDataProtector;
   let dataProtectorConsumer: IExecDataProtector;
-  const DEFAULT_PROTECTED_DATA_DELIVERY_APP =
-    '0x85795d8Eb2B5D39A6E8dfB7890924191B3D1Ccf6';
   let addOnlyAppWhitelist: string;
+  const walletCreator = Wallet.createRandom();
+  const walletConsumer = Wallet.createRandom();
 
   beforeAll(async () => {
-    const walletCreator = Wallet.createRandom();
-    const walletConsumer = Wallet.createRandom();
-
-    // iexecOptions for staging
-    const iexecOptions = {
-      smsURL: 'https://sms.scone-prod.stagingv8.iex.ec',
-      ipfsGatewayURL: 'https://ipfs-gateway.stagingv8.iex.ec',
-      iexecGatewayURL: 'https://api.market.stagingv8.iex.ec',
-      resultProxyURL: 'https://result.stagingv8.iex.ec',
-    };
-
-    const dataProtectorOptions = {
-      iexecOptions: iexecOptions,
-      ipfsGateway: 'https://ipfs-gateway.stagingv8.iex.ec',
-      ipfsNode: 'https://ipfs-upload.stagingv8.iex.ec',
-      subgraphUrl:
-        'https://thegraph-product.iex.ec/subgraphs/name/bellecour/dev-dataprotector-v2',
-    };
-
-    const provider1 = utils.getSignerFromPrivateKey(
-      'https://bellecour.iex.ec',
-      walletCreator.privateKey
-    );
-    const provider2 = utils.getSignerFromPrivateKey(
-      'https://bellecour.iex.ec',
-      walletConsumer.privateKey
-    );
-
     dataProtectorCreator = new IExecDataProtector(
-      provider1,
-      dataProtectorOptions
+      ...getTestConfig(walletCreator.privateKey)
     );
     dataProtectorConsumer = new IExecDataProtector(
-      provider2,
-      dataProtectorOptions
+      ...getTestConfig(walletConsumer.privateKey)
     );
-
     const addOnlyAppWhitelistResponse =
       await dataProtectorCreator.sharing.createAddOnlyAppWhitelist();
     addOnlyAppWhitelist = addOnlyAppWhitelistResponse.addOnlyAppWhitelist;
-  });
 
-  describe('When calling consumeProtectedData() with valid inputs', () => {
+    await createAndPublishWorkerpoolOrder(
```

Comment:
It looks like it doesn't work on my machine, I get:

```
Invalid order signer, must be the resource owner
```

Do you see what could go wrong here?

Reply:
Alright, probably my fault, forgot to run "test:prepare"

Now I get:

```
"Exceeded timeout of 193000 ms for a test.
```
:(